frontend/src/metabase/visualizations/visualizations/PivotTable/utils.ts

```diff
@@ -96,23 +96,28 @@ interface GetLeftHeaderWidthsProps {
   rowIndexes: number[];
   getColumnTitle: (columnIndex: number) => string;
   leftHeaderItems?: HeaderItem[];
-  fontFamily?: string;
+  font?: { fontFamily?: string; fontSize?: string };
```

Comment:
my 2c, we shouldn't type this according to how we test. We always pass `font` as an object in `PivotTable.tsx`, so this shouldn't be optional.

```suggestion
  font: { fontFamily?: string; fontSize?: string };
```